src/Avalonia.Labs.Controls/SwipeView/Swipe.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Threading;
+using System.Windows.Input;
+using Avalonia.Animation;
+using Avalonia.Animation.Easings;
+using Avalonia.Controls;
+using Avalonia.Controls.Presenters;
+using Avalonia.Labs.Controls.Base.Pan;
+using Avalonia.Layout;
+using Avalonia.Markup.Xaml.Templates;
+using Avalonia.Media.Transformation;
+
+namespace Avalonia.Labs.Controls;
+
+public class Swipe : Grid
+{
+    public static readonly StyledProperty<DataTemplate> RightTemplateProperty =
+        AvaloniaProperty.Register<Swipe, DataTemplate>(nameof(Right));
+
+    /// <summary>
+    /// DataTemplate for the right side
+    /// </summary>
+    public DataTemplate Right
+    {
+        get => GetValue(RightTemplateProperty);
+        set => SetValue(RightTemplateProperty, value);
+    }
+
+    public static readonly StyledProperty<DataTemplate> LeftTemplateProperty =
+        AvaloniaProperty.Register<Swipe, DataTemplate>(nameof(Left));
+
+    /// <summary>
+    /// DataTemplate for the left side
+    /// </summary>
+    public DataTemplate Left
+    {
+        get => GetValue(LeftTemplateProperty);
+        set => SetValue(LeftTemplateProperty, value);
+    }
+
+    public static readonly StyledProperty<Control> ContentProperty =
+        AvaloniaProperty.Register<Swipe, Control>(nameof(Content));
+
+    /// <summary>
+    /// The content of the Swipe component
+    /// </summary>
+    public Control Content
+    {
+        get => GetValue(ContentProperty);
+        set => SetValue(ContentProperty, value);
+    }
+
+    public static readonly StyledProperty<SwipeState> SwipeStateProperty =
+        AvaloniaProperty.Register<Swipe, SwipeState>(nameof(SwipeState));
+
+    /// <summary>
+    /// The current state of the Swipe component
+    /// </summary>
+    public SwipeState SwipeState
+    {
+        get => GetValue(SwipeStateProperty);
+        set => SetValue(SwipeStateProperty, value);
+    }
+
+    private readonly ContentPresenter _rightContainer;
+    private readonly ContentPresenter _leftContainer;
+    private readonly ContentPresenter _bodyContainer;
+    private readonly TransformOperationsTransition _transition;
+
+    private double _initialX;
+    private double _currentX;
+    private CancellationTokenSource? _tokenSource;
+    private ICommand? _command;
+    private object? _commandParameter;
```

Comment:
_command and _commandParameter aren't used

Reply:
Fixed.